src/accordion/accordion.ts

```diff
@@ -105,7 +105,7 @@ export interface NgbPanelChangeEvent {
           {{panel.title}}<ng-template [ngTemplateOutlet]="panel.titleTpl?.templateRef"></ng-template>
         </a>
       </div>
-      <div id="{{panel.id}}" role="tabpanel" [attr.aria-labelledby]="panel.id + '-header'" class="card-block" *ngIf="isOpen(panel.id)">
+      <div [id]="panel.id" role="tabpanel" [attr.aria-labelledby]="panel.id + '-header'" class="card-block" *ngIf="isOpen(panel.id)">
```

Comment:
I would rather leave it as-is since the id="{{panel.id}}" form will make sure that we convert whatever is passed to string. Not super-important though.